test/correctness/simd_op_check.cpp

```diff
@@ -187,7 +187,7 @@ class SimdOpCheck : public SimdOpCheckTest {
 
         // SSE 2
 
-        for (int w = 2; w <= 4; w++) {
+        for (int w : {2, 4}) {
```

Comment:
I presume you intend to skip the w=3 case here

Reply:
Yeah, LLVM does different things with different versions, and it's not a case worth testing anyway